lang/uk.po

```diff
@@ -0,0 +1,100 @@
+msgid ""
+msgstr ""
+"Language: uk\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n==1 ? 0 : 1);\n"
+
+msgid "TRANSLATION_AUTHOR\n"
+msgstr "На українську мову переклала yanicka\n"
+
+msgid "FREE_PACKAGES_COUNT\n"
+msgid_plural "FREE_PACKAGES_COUNT\n"
+msgstr[0] "%d вільного пакету (%s)\n"
+msgstr[1] "%d вільних пакетiв (%s)\n"
```

Comment:
Just to make sure - does Ukrainian use only two forms? If you have more plural forms, you can edit line 6 (`Plural-Forms: ...`) to reflect this.

The manual for GNU gettext says that Ukrainian typically uses three plural forms.
https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html

Reply:
I just saw the same "mistake" in the Russian translation ru.po. I edited it too.